woocommerce-services.php

```diff
@@ -845,6 +843,11 @@ public function rest_api_init() {
 			$this->set_rest_self_help_controller( $rest_self_help_controller );
 			$rest_self_help_controller->register_routes();
 
+			require_once __DIR__ . 'classes/class-wc-rest-connect-service-data-refresh-controller.php' ) );
```

Comment:
@frosso, looks like a syntax error slipped in due to a merge. I think the trailing `) )` need to be removed.

Reply:
Thanks! Approved https://github.com/Automattic/woocommerce-services/pull/2350